### DISCUSSION TITLE
Hello everyone!

### DISCUSSION THREAD

**Discussion by ghost** [OpenMined/courses#146] [Introduce Yourself]
Hello, guys! I am Tofig Nazarbayov from Baku, Azerbaijan. I am oil and gas industry guy :D. 

I did my bachelor's degree in the field of Geosciences at French-Azerbaijani University - a collaborative university offering 2 diplomas of the University of Strasbourg and Azerbaijan State Oil and Industry University. Now I pursue master's degree in the field of Geosciences at the same university. I speak English, Russian, French, and Azeri.

I participated in different programs and internships in various oil companies and realized that big data privacy is a really important aspect of the oil and gas industry, the importance of which is quite often underestimated. There is a lot of data in this industry, and it is not always correctly stored, controlled, and curated. I started learning machine learning about 2 years ago. I think that extensive usage of machine learning and attention to data privacy are crucial for the Geophysical and Petroleum fields.

Very excited about this course! Nice to meet all of you!

And here is my [Linkedin](https://www.linkedin.com/in/tofignazarbayov/)

**Comment by yemikifouly** [comment]
Hello Tofnaz! 👋 Nice to meet you and welcome to the discussion board! 

You know, I have never, ever thought of the geophysical and petroleum fields as being sensitive fields when it came to privacy. You know, when we think about data and privacy it is easy to think about health data and similar data sets but the petroleum filed, I have never considered. This is so interesting. 

If you don't mind, could you please share the type of data that is often stored in your field and how it not being correctly controlled can be detrimental? It is my curiosity talking. 😃 Also, I hope you are enjoying the course very much.

**Reply by ghost** [reply]
It will be a little bit long, sorry for that :D

The data, that may pose some danger if leaked to the public is, for example, geotechnical data. Geotechnical companies often are hired by big operator companies to lay cables, pipes, and other types of infrastructure to facilitate the operator's workflow and create necessary conditions for the operations. As they do bathymetry of the seabed, study morphology, and etc., they build the best trajectory to build this pipe/cable/whatever it is. They keep all the geographical coordinates data, the bathymetry data, the sonar of the seabed, thus, describing the trajectory and its positioning in very small details. And 99% of the time the pipe/cable is built strictly following this data.

So, such kind of data (position of the oil or gas pipeline, or electricity cable) can be considered strategic objects by the government. It is true because if this information gets into the wrong hands, huge damage can be done to people, especially to those who work on offshore platforms (hundreds of kilometers away from land) connected to these cables. And, unfortunately, I wouldn't say that the privacy of this data is very well preserved. At least, that's what I noticed in my country.

But I hope, that I will learn many new things from your course, and will be able to bring the new privacy strategies to the table for the oil and gas industry!

**Reply by yemikifouly** [reply]
Oh wow!!! Thanks for the great explanation! By the beginning of the 3rd paragraph I was already starting to see how that data is extremely sensible data even though it is not personal data. Actually, that data in the wrong hands can quickly become a matter of national and even international security. How very interesting Tofnaz! Thank you again for explaining. 

How far along are you with the course? Did you get introduced yet to concepts you think interesting? 🙂 You definitely will though. The course has a lot to offer.

**Reply by ghost** [reply]
I am not that far, just on Lesson 2 at "Data is Fire" part. I have final exams at my university now, so, unfortunately, this slows down my progress on this course. 

I am enjoying the course, I liked the discussion about contextual integrity. It was presented by Emma, if I am not mistaken. The example with facial recognition is great! I never really thought deeply about the idea that face can be considered private or public data, depending on the context of the information flow. 

Exciting course!
